README.md

```diff
@@ -42,4 +40,32 @@ The JVM will then listen for a debugger to be attached on port 8189.  The JVM wi
 the debugger to be attached before starting Echo; the relevant JVM arguments can be seen and
 modified as needed in `build.gradle`.
 
-[//]: # "Only here to retrigger the echo build"
+## Configuring
+echo can run in two modes: in-memory and SQL.
+
+
+**In-memory** mode keeps all CRON trigger information in RAM. While this is simpler to configure (this is the default) the in-memory mode doesn't provide for any redundancy because it requires that a single instance of echo scheduler be running. If/when that instance goes down, CRON triggers will not fire.
+
+**SQL** mode keeps all CRON trigger information in a single SQL database. This allows for multiple echo scheduler instances to run providing redundancy (only one instance will trigger a given CRON).
+
+For SQL you will need to initialize the database, an initialization script for MySql is provided [here]()
+
+### Missed CRON scheduler
+
+
+### Configuration options
+
+`scheduler.pipelineConfigsPoller.enabled`
+
+`scheduler.compensationJob.enabled`
+
+`scheduler.enabled`
+
+### Sample deployment
+Here are examples of what configurations you can deploy `echo`.
+
+|                   | Using in-memory           | Using SQL |
+|-------------------|---------------------------|-----------|
+|**Server Group 1** |3x `echo`                  | 3x `echo` with `echo-scheduler`
+|**Server Group 2** |1x `echo-scheduler`        | 1x `echo-missed-scheduler`
+|**Server Group 3** |1x `echo-missed-scheduler` | n/a
```

Comment:
I don’t think the SQL version would need the missed scheduler cluster either? Quartz automatically handles misfires! Worth testing to see if we get similar behavior (we should).

Reply:
well, there is a caveat... Our "missfire" handling is way more elaborate... e.g it will only refire if the misfire is within a given window (which I believe we can configure with quartz). But we also actually ask orca if the execution happened, so in theory the missed scheduler will catch other random failures not just quartz misfires... 
For now, I think we keep it as is (as in, keep the `MissedPipelineTriggerCompensationJob` - since it's controlled by property anyway) After it runs in our env, we can talk through if we really need it - would certainly simplify the operation of `echo` if it's gone
@dreynaud will have opinions for sure so adding him

Reply:
Happy to experiment but yes, let's make sure we have parity with the current missed scheduler before pulling the plug

Reply:
Ok. With that in mind, we might want to disable quartz performing misfire remediation entirely. I’d be keen to see if we can provide custom logic for how quartz handles misfires. I don’t like maintaining additional stacks; one of the objectives of this quartz work is to reduce operational overhead. Let’s see if:

1. Quartz can support custom behavior for resolving misfires.
2. If not, evaluate how we can roll our custom behavior into the single stack.

Reply:
It looks like a `TriggerListener` could potentially do the trick for us.

Reply:
yes, we already have a trigger listener - and it always instructs quartz not to "re-fire" but yes, in a subsequent PR I will address the quartz/missfire/missed pipeline issue